examples/base-vite-react-tailwind-ts/README.md

```diff
@@ -1,4 +1,10 @@
-# Base UI - Create React App example with Tailwind CSS in TypeScript
+# Base UI - Vite React example with Tailwind CSS in TypeScript
```

Comment:
```suggestion
# Base UI - Vite.js example with Tailwind CSS in TypeScript
```

to match with https://github.com/mui/material-ui/tree/master/examples/base-vite-tailwind